canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -38,28 +39,28 @@
 	.properties-radioset-panel {
 		margin-bottom: $spacing-02;
 		margin-right: $spacing-04;
-		display: grid;
-		grid-template-columns: 1fr 8fr;
-		grid-template-rows: auto;
-		grid-template-areas:
-			"radio tooltip"
-			"panel panel";
+		display: flex;
+		justify-content: flex-start;
+		align-items: center;
+		flex-wrap: wrap;
+		overflow: hidden;
 		.bx--radio-button-wrapper{
-			grid-area: radio;
 			margin-right: 0; // We removed the space between radio & tooltip to position it correctly
 		}
-		.tooltip-container{
-			grid-area: tooltip;
+		.properties-radioset-tooltip {
+			display: flex;
+			align-items: center;
 		}
 		.properties-control-nested-panel{
-			grid-area: panel;
+			flex-basis: 100%;
+			width: 100%;
 		}
 		&:last-of-type {
 			margin-bottom: 0; // don't want margin when last radio button in group
 		}
-
 		.bx--radio-button-wrapper > label {
 			justify-content: left; // override carbon style to center justify radio-button labels
+			margin:  2.3px 0 0 $spacing-01;
```

Comment:
Why 2.3px?  We shouldn't need to use specific spacing values like this.

Reply:
I don't think this is needed at all.  Carbon should be setting this correctly already.

Reply:
You might need to add `align-items: center;` to center the label.  Then set a margin of 1.5px on `properties-radioset-tooltip`.  That fixes the selection issue.

Reply:
`margin: 2.5px 0 1.5px 1.5px;` on `properties-radioset-tooltip `fixed the selection issue